client/controllers/incidentReports/incidentSecondaryDetails.coffee

```diff
@@ -0,0 +1,30 @@
+Template.incidentSecondaryDetails.onCreated ->
+  @detailsOpen = new ReactiveVar(false)
+  Meteor.defer =>
+    @$('[data-toggle=tooltip]').tooltip delay: show: '400'
+
+Template.incidentSecondaryDetails.helpers
+  detailsOpen: ->
+    Template.instance().detailsOpen.get()
+
+  firstLocationName: ->
+    firstLocation = Template.instance().data.locations[0]
+    firstLocation?.countryName or
+      firstLocation?.admin1Name or
+      firstLocation?.admin2Name or
+      firstLocation?.name
```

Comment:
Every location has a name so you can always use that. Is there a reason for using the country name instead? I don't think the admin names will be defined if there is no country name, so you could leave those out.